docs/installation.md

```diff
@@ -0,0 +1,110 @@
+# Installation
+## Manually
+> **Note**: We do not recommend this method if you want to get auto-updating from package managers or have more platforms supported.
+### X86_64 or ARM64 (M1/M2) Linux / macOS
+```bash
+curl -s https://raw.githubusercontent.com/kubescape/kubescape/master/install.sh | /bin/bash
+```
+
+### X86_64 Windows
+You must have PowerShell v5.0 or higher:
+```powershell
+iwr -useb https://raw.githubusercontent.com/kubescape/kubescape/master/install.ps1 | iex
+```
+
+If you get an error, you may need to change the execution policy:
+```powershell
+Set-ExecutionPolicy RemoteSigned -scope CurrentUser
+```
+
+## openSUSE
+> **Note**: openSUSE community-supported.

```

Comment:
Isn't this managed by us?
The "community supported" label means it's not built or updated by our workflow.  Most of the rest of the things we're adding are built by us, so they shouldn't be marked as such.

Reply:
No, it's not. The installation method works with https://build.opensuse.org/package/show/devel:kubic/kubescape , and that one is managed by openSUSE community and not built or updated by our workflow.